cdap-proto/src/main/java/io/cdap/cdap/proto/ErrorClassificationResponse.java

```diff
@@ -32,10 +32,13 @@ public final class ErrorClassificationResponse {
   private final String errorCode;
   private final String supportedDocumentationUrl;
   private transient final String throwableClassName;
+  private transient final Integer rulePriority;
+  private transient final String ruleId;
```

Comment:
Why are you adding transient things to the response? If you don't want to send it to user just don't include it. It's a proto, it's better not to put unneeded things here

Reply:
These things are not meant to be returned in the response but I still want to add it in the internal metrics for analysis of data.

Reply:
Created a wrapper and moved these fields there.